uber/automated_emails.py

```diff
@@ -14,14 +14,16 @@ class AutomatedEmail:
     }
     extra_models = queries  # we've renamed "extra_models" to "queries" but are temporarily keeping the old name for backwards-compatibility
 
-    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
+    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True, email_id=None):
         self.model, self.template, self.needs_approval = model, template, needs_approval
         self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)
         self.cc = cc or []
         self.bcc = bcc or []
         self.extra_data = extra_data or {}
         self.sender = sender or c.REGDESK_EMAIL
         self.instances[self.subject] = self
+        self.email_id = email_id or self.subject
+        self.email_id = self.email_id.format(EVENT_NAME=c.EVENT_NAME)
```

Comment:
Yeah, the point of using the subject line at all (instead of just adding IDs for all the emails or creating automatic idents in another way) was so we WOULDN'T have to do a data migration. I suppose we'll still have to migrate the schema, though, since we're adding a column to an existing table.